### PR TITLE
The YAML markup language is called "YAML", not "YML"

### PR DESCRIPTION
**Pull request by adrienverge**
As [someone said](https://github.com/ceph/s3-tests/commit/e17c56a) on the internet:

    Say ".yaml" not ".yml".
    This is not MS-DOS, and YML is a Yahoo XML dialect.

Similarly, we use 'json', not 'jsn'.
This commit renames the `yml` bear directory to `yaml`.

References:
https://github.com/ceph/s3-tests/commit/e17c56a
https://github.com/adrienverge/yamllint/commit/41733fc
http://yaml.org/spec/1.2/spec.html

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

### CODE REVIEW

File: tests/yml/RAMLLintBearTest.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from bears.yml.RAMLLintBear import RAMLLintBear
+from bears.yaml.RAMLLintBear import RAMLLintBear
```
Comment (meetmangukiya) [inline tests/yml/RAMLLintBearTest.py]: Why not rename the test directory as well, then ?

Comment (meetmangukiya) [inline tests/yml/RAMLLintBearTest.py]: Also as gitmate said, please reduce the length of shortlog to 50 chars

Comment (adrienverge) [inline tests/yml/RAMLLintBearTest.py]: @meetmangukiya Thanks for reviewing.

> Why not rename the test directory as well, then ?

Sure!

> Also as gitmate said, please reduce the length of shortlog to 50 chars

It is already, the comment refers to an older commit.

**Comment by adrienverge** [comment]
Updated.

**Comment by jayvdb** [comment]
There is also `tests/ruby/RuboCopBearTest.py` and `tests/swift/TailorBearTest.py` which use `tests/ruby/test_files/rubocop_config.yml` and `tests/swift/test_files/tailor.yml` respectively.  Those need renaming.

`.ci/appveyor.yml` can also be renamed, but that should be a separate PR.
`.travis.yml` cant be renamed, as far as I know.
`.codecov.yml` is more likely able to be renamed; if not, raise a bug upstream - the author is very likely to be receptive.
I dont know about `circle.yml`

**Comment by adrienverge** [comment]
@jayvdb thanks for having a look here!

I intentionally didn't touch file extensions as they aren't directly related to this change (which is about the language name).

Since you raise this point, I updated this PR for `rubocop_config.yaml` and `tailor.yaml`.

About CI config files, you're right: if done, it should be in a separated PR.

**Comment by jayvdb** [comment]
ack 4a101b7

**Comment by jayvdb** [comment]
ping @meetmangukiya

**Comment by sils** [comment]
@rultor merge

**Comment by sils** [comment]
Hey @adrienverge thanks for your contribution! Do you have any idea on how the contribution process or using coala could have been smoother? (See also coala.io/usability)

**Comment by sils** [comment]
@adrienverge FYI rultor will release it as a prerelease so you'll be able to use it right now if you install coala with `--pre`

**Comment by adrienverge** [comment]
@sils The contribution process was pretty smooth actually. Thanks!

**Pull request merged**